incubator/gogs/templates/configmap.yaml

```diff
@@ -0,0 +1,88 @@
+---
```

Comment:
Can you remove these delimiters from your templates? While they *should* be valid there's an issue with helm that causes problems deleting resources when you delete a release.